packages/data-context/src/sources/ProjectConfigDataSource.ts

```diff
@@ -32,7 +32,10 @@ export class ProjectConfigDataSource {
       this.ctx.coreData.currentProject.config = Promise.resolve().then(async () => {
         const configFile = await this.ctx.config.getDefaultConfigBasename(projectRoot)
 
-        return this.ctx._apis.projectApi.getConfig(projectRoot, { configFile })
+        return this.ctx._apis.projectApi.getConfig(projectRoot, {
```

Comment:
I don't have context on this part of the code base at this very moment, but before merging let's find out

1. Is `testingType` **ever** null here? If not, let's `throw Error('TestingType is null')
2. If is **is** valid, I think now is a good time to add at least a basic unit or integration test?

Reply:
All valid, I think we can cover this in an e2e test. It's getting late so I can check in tomorrow.